system-variables.md

```diff
@@ -759,28 +812,40 @@ Constraint checking is always performed in place for pessimistic transactions (d
 > - Since v5.4.0, for a newly deployed TiDB cluster, this variable is enabled by default.
 
 - Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
 - Default value: `ON`
 - This variable is used to control whether to enable the index merge feature.
 
+### tidb_enable_legacy_instance_scope <span class="version-mark">New in v6.0.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Default value: `ON`
+- This variable permits `INSTANCE` scoped variables to be set with `SET SESSION` as well as `SET GLOBAL` syntax.
+- This option is enabled by default for compatibility with earlier versions of TiDB.
+
 ### tidb_enable_list_partition <span class="version-mark">New in v5.0</span>
 
 > **Warning:**
 >
 > Currently, List partition and List COLUMNS partition are experimental features. It is not recommended that you use it in production environments.
 
 - Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
 - Default value: `OFF`
 - This variable is used to set whether to enable the `LIST (COLUMNS) TABLE PARTITION` feature.
 
-### `tidb_enable_mutation_checker` (New in v6.0.0)
+### tidb_enable_mutation_checker <span class="version-mark">New in v6.0.0</span>
 
 - Scope: SESSION | GLOBAL
-- Default value: `ON`
+- Persists to cluster: Yes
+- Default value: `OFF`
```

Comment:
The default is wrong. It is OFF not on. See https://github.com/pingcap/tidb/pull/31547 by @[ekexium](https://github.com/ekexium)

(It is wrong in 6.0 and master)